app/vmctl/prometheus.go

```diff
@@ -47,7 +55,6 @@ func (pp *prometheusProcessor) run() error {
 	if err := barpool.Start(); err != nil {
 		return err
 	}
-	defer barpool.Stop()
```

Comment:
Why this was replaced with direct call to `.Stop` at the end of the function?
As far as I can see, there are code paths which return errors so a `barpool` will not be stopped in case of an error.

Reply:
There was an issue with show stats and end of barpool i fixed it by moving logic to the function. Now it works correctly.

<img width="1718" alt="Screenshot 2025-01-02 at 12 33 03" src="https://github.com/user-attachments/assets/ca11874f-e4c4-43be-a5b2-07e03de1be7e" />

Reply:
How it works now
<img width="1720" alt="Screenshot 2025-01-02 at 12 37 27" src="https://github.com/user-attachments/assets/fbedb0cd-bb49-4161-8154-81aab1a54b35" />